### ISSUE TITLE
[vk.com] Authentication issue

### ISSUE THREAD

**Issue by spvkgn** [rg3/youtube-dl#4967]
vk.com has issue with authentication, can you please fix it
```
~ $ youtube-dl -v -u USER -p PASS https://vk.com/video189477881_170900782?hash=7e0b5e56106b940f
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', '-u', u'PRIVATE', '-p', u'PRIVATE', 'https://vk.com/video189477881_170900782?hash=7e0b5e56106b940f']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.16.1
[debug] Python version 2.7.3 - Linux-3.2.0-69-generic-pae-i686-with-LinuxMint-13-maya
[debug] exe versions: ffmpeg 2.4.2, ffprobe 2.4.2, rtmpdump 2.4
[debug] Proxy map: {}
[vk.com] Logging in as USER
[vk.com] 189477881_170900782: Downloading webpage
ERROR: Unable to extract vars; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 633, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 270, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vk.py", line 184, in _real_extract
    data_json = self._search_regex(r'var vars = ({.*?});', info_page, 'vars')
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 532, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract vars; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by yousufalim** [comment]
yup same


[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://vk.com/video-87482294_171259320', '-u', u'PRIVATE', '-p', u'PRIVATE']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.16
[debug] Python version 2.7.5 - Linux-3.10.23-xxxx-std-ipv6-64-x86_64-with-fedora-20-Heisenbug
[debug] exe versions: ffmpeg 2.1.5, ffprobe 2.1.5
[debug] Proxy map: {}
[vk.com] Logging in as admin@pixup.us
[vk.com] -87482294_171259320: Downloading webpage
ERROR: Unable to extract vars; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type youtube-dl -U to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 633, in extract_info
ie_result = ie.extract(url)
File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 270, in extract
return self._real_extract(url)
File "/usr/local/bin/youtube-dl/youtube_dl/extractor/vk.py", line 184, in _real_extract
data_json = self._search_regex(r'var vars = ({.*?});', info_page, 'vars')
File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 532, in _search_regex
raise RegexNotFoundError('Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract vars; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type youtube-dl -U to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

**Comment by dstftw** [comment]
Will be fixed in the next version.

**Comment by phihag** [comment]
Fixed in youtube-dl 2015.03.03.1. See [our FAQ](https://yt-dl.org/update) if you need help updating.
